DQM/SiPixelMonitorClient/src/SiPixelDaqInfo.cc

```diff
@@ -42,10 +41,8 @@ void SiPixelDaqInfo::dqmEndLuminosityBlock(DQMStore::IBooker &iBooker,
     firstLumi = false;
   }
 
-  if (auto runInfoRec = iSetup.tryToGet<RunInfoRcd>()) {
+  if ( edm::ESHandle<RunInfo> sumFED = iSetup.getHandle(runInfoToken_) ) {
```

Comment:
@makortel, is this the right way to do it since in the updated version there is no check that the corresponding Record is present?

Reply:
The `edm::EventSetup::getHandle()` will throw an exception if the `RunInfoRcd` is missing. Therefore, if your intention is to preserve the behavior, you should leave the `edm::EventSetup::tryToGet()` call, and get the product from the Record with the token, e.g.
```cpp
const RunInfo& sumFED = runInfoRec->get(runInfoToken_);
```

Reply:
@makortel, thanks for the tip.